src/_html/how_we_dive.html

```diff
@@ -18,7 +18,7 @@
             "gallery": "./gallery.html",
             "contact_us": "./contact_us.html"
         },
-        "title": "How we <br>diving",
```

Comment:
I asked the boss and she told that she want make it in two strings. So I think, we return all "br". And we should a little reduce margin in button, cos text will overlay button